mifosng-android/src/main/java/com/mifos/mifosxdroid/online/datatablelistfragment/DataTableListFragment.java

```diff
@@ -241,7 +245,12 @@ public void onClick(View v) {
             }
         });
     }
-
+    public void hideKeyboard(View view) {
```

Comment:
It would be much better if you make this method generic by getting `view` and `context` as parameters and add it `Utils.java`, in future you or other developers also can use it on different screens.

Reply:
@jawidMuhammadi  i have done the changes..please review it :)

Reply:
@neeraj0403 are you sure that you have pushed your code. I don't see any changes.